quickstart/nodejs/redistest.js

```diff
@@ -1,41 +1,50 @@
 var redis = require("redis");
+require('dotenv').config()
+
+// Environment variables for cache
+const cacheHostName = process.env.AZURE_CACHE_FOR_REDIS_RESOURCE_NAME;
+const cachePassword = process.env.AZURE_CACHE_FOR_REDIS_RESOURCE_KEY;
+
+if(!cacheHostName) throw Error("cacheHostName is empty")
```

Comment:
Should these errors mention the the actual environment variable names?

Reply:
Sure I'll fix.